app/stacks/uk-west/applications-service/secrets.tf

```diff
@@ -27,35 +27,66 @@ resource "azurerm_key_vault_secret" "applications_app_insights_connection_string
   tags = local.tags
 }
 
-resource "azurerm_key_vault_secret" "applications_sql_server_password" {
+resource "azurerm_key_vault_secret" "applications_sql_server_password_admin" {
   #checkov:skip=CKV_AZURE_41
 
   content_type = "text/plain"
   key_vault_id = var.key_vault_id
-  name         = "applications-service-sql-server-password"
+  name         = "applications-service-sql-server-password-admin"
   value        = azurerm_mssql_server.applications_sql_server.administrator_login_password
 
   tags = local.tags
 }
 
-resource "azurerm_key_vault_secret" "applications_sql_server_username" {
+resource "azurerm_key_vault_secret" "applications_sql_server_password_app" {
   #checkov:skip=CKV_AZURE_41
 
   content_type = "text/plain"
   key_vault_id = var.key_vault_id
-  name         = "applications-service-sql-server-username"
+  name         = "applications-service-sql-server-password-app"
+  value        = random_password.applications_sql_server_password_app.result
+
+  tags = local.tags
+}
+
+resource "azurerm_key_vault_secret" "applications_sql_server_username_admin" {
+  #checkov:skip=CKV_AZURE_41
+
+  content_type = "text/plain"
+  key_vault_id = var.key_vault_id
+  name         = "applications-service-sql-server-username-admin"
   value        = azurerm_mssql_server.applications_sql_server.administrator_login
 
   tags = local.tags
 }
 
-resource "azurerm_key_vault_secret" "applications_sql_server_connection_string" {
+resource "azurerm_key_vault_secret" "applications_sql_server_username_app" {
+  #checkov:skip=CKV_AZURE_41
+
+  content_type = "text/plain"
+  key_vault_id = var.key_vault_id
+  name         = "applications-service-sql-server-username-app"
+  value        = local.sql_server_username_app
+}
+
+resource "azurerm_key_vault_secret" "applications_sql_server_connection_string_admin" {
+  #checkov:skip=CKV_AZURE_41
+
+  content_type = "text/plain"
+  key_vault_id = var.key_vault_id
+  name         = "applications-service-sql-server-connection-string-admin"
+  value        = local.sql_connection_string_admin
+
+  tags = local.tags
+}
+
+resource "azurerm_key_vault_secret" "applications_sql_server_connection_string_app" {
   #checkov:skip=CKV_AZURE_41
 
   content_type = "text/plain"
   key_vault_id = var.key_vault_id
-  name         = "applications-service-sql-server-connection-string"
-  value        = local.sql_connection_string
+  name         = "applications-service-sql-server-connection-string-app"

```

Comment:
The app-service references the old connection-string secret by name; this will break that. 

is it worth applying this to Applications separately from BO?

Reply:
Thanks, that's updated now. I figure both need changing so may as well do them together.